### PR TITLE
Update voice_registry_system.md: tomorrow's ecosystem, first thoughts on value propositions

### PR DESCRIPTION
**Pull request by jcstine**
Recommended additions to the VRS master document.

### CODE REVIEW

File: components/voice_registry_system.md
Comment (pmotch) [inline components/voice_registry_system.md:106]: I think this needs to be `1.0.4` and the subsequents sections below incremented.

File: components/voice_registry_system.md
Comment (pmotch) [inline components/voice_registry_system.md:111]: I suggest using ordered list bullet markdown for those numbered things, for readability.

**Review by pmotch — COMMENTED** [review]
Not requesting changes as I don't want to be a bottleneck to getting this merged. I'll defer to Maria given the copy. Just noticed a few tweaks and wanted to call it out.

File: components/voice_registry_system.md
Comment (pmotch) [reply components/voice_registry_system.md]: Looks like Trammell took care of it 👍

File: components/voice_registry_system.md
Comment (pmotch) [reply components/voice_registry_system.md:106]: This PR looks good to go once this is sorted out ^.

Comment (maridob) [reply components/voice_registry_system.md:106]: I agree

Comment (pmotch) [reply components/voice_registry_system.md:106]: The section numbering still appears to be an issue. Once this is fixed, we can merge this PR.

Comment (JohnTrammellTarget) [reply components/voice_registry_system.md:106]: I think I disagree; 1.0.3.2 follows naturally from 1.0.3.1 and should be a sibling in the outline.

**Review by JohnTrammellTarget — APPROVED** [review]
👍

**Comment by Elizabeth-Robins** [comment]
No comments

**Pull request merged**
